bin/verify-indent

```diff
@@ -1,7 +1,9 @@
-#!/bin/sh
+#!/bin/bash
 
 sh indent.sh
 
+git diff --quiet
```

Comment:
Finally found a fix. I have no idea why repeating the command here makes it work. Any guesses?